source/deploy/standalone/proxy_ssl.md.erb

```diff
@@ -0,0 +1,247 @@
+ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES256-SHA384:ECDHE-RSA-AES128-SHA:\
+ECDHE-ECDSA-AES256-SHA384:ECDHE-ECDSA-AES256-SHA:ECDHE-RSA-AES256-SHA:\
+DHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA:DHE-RSA-AES256-SHA256:DHE-RSA-AES256-SHA:\
+ECDHE-ECDSA-DES-CBC3-SHA:ECDHE-RSA-DES-CBC3-SHA:EDH-RSA-DES-CBC3-SHA:\
+AES128-GCM-SHA256:AES256-GCM-SHA384:AES128-SHA256:AES256-SHA256:AES128-SHA:\
+AES256-SHA:DES-CBC3-SHA:!DSS
+    SSLCertificateFile "/private/etc/apache2/server.crt"
+    SSLCertificateKeyFile "/private/etc/apache2/server.key"
+    SSLCompression off # not all versions of apache+openssl allow this option
+
+    Header always edit Set-Cookie ^(.*)$ $1;HttpOnly;Secure
+    Header always set Strict-Transport-Security "max-age=31536000; includeSubDomains"
+
+    ServerSignature Off
+
+&lt;/VirtualHost&gt;
+</pre>
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Nginx
+            </td>
+            <td>
+                Change your listen configs from port 80 to the ssl verisons to the right. create a redirect from http to https as shown. Add the ssl configs as described to the right. You must generate a dhparam file, this can be done with <code><span class="prompt">$ </span>openssl dhparam -out /usr/local/etc/nginx/ssl/dhparam.pem 2048</code>
+            </td>
+            <td>
+<pre class="highlight nginx">
+server_tokens off;
+server {
+listen 80 default_server;
+listen [::]:80 default_server;
+return 301 https://$host$request_uri;
+}
+
+server {
+listen 443 ssl;
+listen [::]:443 ssl;
+
+ssl_certificate /usr/local/etc/nginx/ssl/server.crt;
+ssl_certificate_key /usr/local/etc/nginx/ssl/server.key;
+ssl_session_timeout 1d;
+ssl_session_cache shared:SSL:50m;
+ssl_session_tickets off;
+ssl_dhparam /usr/local/etc/nginx/ssl/dhparam.pem;
+ssl_prefer_server_ciphers on;
+ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
+ssl_ciphers 'ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:
+ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:
+ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:
+DHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256:
+ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES256-SHA384:ECDHE-RSA-AES128-SHA:
+ECDHE-ECDSA-AES256-SHA384:ECDHE-ECDSA-AES256-SHA:ECDHE-RSA-AES256-SHA:
+DHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA:DHE-RSA-AES256-SHA256:DHE-RSA-AES256-SHA:
+ECDHE-ECDSA-DES-CBC3-SHA:ECDHE-RSA-DES-CBC3-SHA:EDH-RSA-DES-CBC3-SHA:
+AES128-GCM-SHA256:AES256-GCM-SHA384:AES128-SHA256:AES256-SHA256:AES128-SHA:
+AES256-SHA:DES-CBC3-SHA:!DSS';
+
+add_header Strict-Transport-Security max-age=31536000;
+location / {
+proxy_pass http://localhost:3000;
+}
+}
+</pre>
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Passenger Standalone
+            </td>
+            <td>
+                No changes needed.
+            </td>
+            <td>
+                <pre class="highlight"><span class="prompt">$ </span>passenger start</pre>
```

Comment:
also, I don't understand how just starting passenger (default) has anything to do with an ssl terminating reverse proxy?